sample-apps/monaibundle/README.md

```diff
@@ -50,9 +69,6 @@ monailabel start_server --app workspace/monaibundle --studies workspace/images -
 # Pick DeepEdit And Segmentation model (multiple models)
 monailabel start_server --app workspace/monaibundle --studies workspace/images --conf models "spleen_ct_segmentation_v0.1.0,spleen_deepedit_annotation_v0.1.0"
```

Comment:
is it still spleen_ct_segmentation_v0.1.0 or spleen_ct_segmentation

Reply:
It's should be with version "_v0.1.0" or "_v0.1.1", users need to specify which bundle version to load.

Reply:
Hi @tangy5 ,

Maybe you can leverage the MONAI bundle `download` API with `bundle name` and `verson`:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/scripts.py#L175

Thanks.